archive/src/main/java/io/orkes/conductor/dao/postgres/archive/PostgresArchiveDAO.java

```diff
@@ -52,22 +52,33 @@ public class PostgresArchiveDAO extends PostgresBaseDAO implements ArchiveDAO, D
     public static final String UNKNOWN = PostgresArchiveDAO.class.getName() + "_UNKNOWN";
     public static final String GET_WORKFLOW_FAMILY = "WITH RECURSIVE workflow_hierarchy AS (" +
             "   SELECT" +
-            "       workflow_id, parent_workflow_id, json_data, status from archive.workflow_archive where workflow_id = ?" +
+            "       workflow_id, parent_workflow_id, json_data, status from archive.workflow_archive where workflow_id = ? " +
             "   UNION ALL" +
-            "   SELECT child_wf.workflow_id workflow_id, child_wf.parent_workflow_id, child_wf.json_data, child_wf.status" +
+            "   SELECT child_wf.workflow_id workflow_id, child_wf.parent_workflow_id, child_wf.json_data, child_wf.status " +
```

Comment:
these trailing spaces should not be committed

Reply:
removed